zebra-state/src/memory_state.rs

```diff
@@ -575,7 +575,11 @@ mod tests {
     fn forked_equals_pushed() -> Result<()> {
         zebra_test::init();
 
-        proptest!(|((chain, count) in arbitrary_chain_and_count())| {
+        proptest!(ProptestConfig::with_cases(env::var("PROPTEST_CASES")
+                                          .ok()
+                                          .and_then(|v| v.parse().ok())
+                                          .unwrap_or(1)),
```

Comment:
Oh my, 1 case.

Let's make a tracking item or ticket to look into proptest speed?

Reply:
good idea. I have an idea for it already so I'll document that in the issue RN as I create it.